gossipd/gossip_store.c

```diff
@@ -587,8 +587,8 @@ bool gossip_store_compact(struct gossip_store *gs)
 }
 
 u64 gossip_store_add(struct gossip_store *gs, const u8 *gossip_msg,
-		     u32 timestamp, bool push, bool spam,
-		     const u8 *addendum)
+		     u32 timestamp, bool push,
+		     bool spam, const u8 *addendum)
 {
```

Comment:
This commit needs to be quashed into previous.  Or, TBH, simply removed as not worthwhile.